src/app/(nested-pages)/blog/page.tsx

```
@@ -1,8 +1,17 @@
-import Link from 'next/link';
+import type { Metadata } from "next";
+import Link from "next/link";
 
 type Props = {};
 
-export default function BlogPage({}: Props) {
+export const metadata: Metadata = {
+  title: "Blog",
+};
+export default async function BlogPage({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)